addons/web/static/src/views/kanban/kanban_view.scss

```diff
@@ -0,0 +1,66 @@
+.o-dropdown--kanban-record-menu {
```

Comment:
I said kanban_view.scss but I meant kanban_controller.scss, where the rest of the style is defined...